### ISSUE TITLE
How do you think about this library versus Azure's python API for data lake? 

### ISSUE THREAD

**Issue by alexlenail** [michaelbironneau/pyadl#1]
Although it's undocumented, Azure has built a [python sdk for azure data lake](https://github.com/Azure/azure-sdk-for-python/blob/master/azure-mgmt-datalake-store/azure/mgmt/datalake/store/filesystem/operations/file_system_operations.py). I don't fully understand how this is different from that, and I'd like to know whether you would recommend using one over the other and why.

**Comment by michaelbironneau** [comment]
Thanks for that link, I didn't know Microsoft were working on that. Based on the history they started working on it shortly after I wrote this. It makes calls to the ADL's WebHDFS-compliant REST API under the hood and so it supports the same operations as pywebhdfs. However, if you're already using pywebhdfs in another project then my client will be a drop-in replacement whereas Microsoft names their methods differently so you would need to rewrite your code. It's also not clear to me whether Microsoft's client would work on a different WebHDFS backend or if there's any ADL-specific stuff in there.

In other words, if you're already using pywebhdfs and you don't want to rewrite your code then use my client, otherwise probably best to use Microsoft's (un)official one.

**Comment by alexlenail** [comment]
Thanks!
